src/libraries/System.Net.WebSockets/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -1464,7 +1459,7 @@ private void ReleaseSendBuffer()
         }
 
         private static int CombineMaskBytes(Span<byte> buffer, int maskOffset) =>
-            BitConverter.ToInt32(buffer.Slice(maskOffset));
+            BinaryPrimitives.ReadInt32LittleEndian(buffer.Slice(maskOffset));
```

Comment:
I don't believe this is correct here. The mask is just 4 bytes which were packed as int in the old implementation just for storage.